packages/core/src/utils.ts

```diff
@@ -40,7 +42,9 @@ export function block({
 	return () => {
 		input.off('keypress', clear);
 		if (hideCursor) process.stdout.write(cursor.show);
-		if (input.isTTY) input.setRawMode(false);
```

Comment:
I would recommend removing both references to `input.setRawMode` since it it's not required for clearing the screen and checking for sigint.